.circleci/config.yml

```diff
@@ -0,0 +1,67 @@
+version: 2
+
+jobs:
+  test:
+    docker:
+      - image: circleci/ruby:2.4.2
+    steps:
+      - checkout
+      - run:
+          name: Install env dependencies
+          command: |
+            sudo apt-get update
+            sudo apt-get install postgresql-client cmake pkg-config
```

Comment:
Does circlemator need these?

Reply:
Yup:
- `postgresql-client` is required by `pg` gem
- `cmake` and `pkg-config` are required by `rugged`

We also need to change the `required` check before merging this

Reply:
What's using postgresql though? :thinking:

Reply:
I've removed the required check. Ping me on slack when it's merged and I'll enable the new check.